openmetadata-ui/src/main/resources/ui/src/pages/TagPage/TagPage.tsx

```diff
@@ -0,0 +1,680 @@
+                  entityType={EntityType.TAG}
+                  hasPermission={editTagsPermission}
+                />
+              ) : null,
+              ...COMMON_RESIZABLE_PANEL_CONFIG.RIGHT_PANEL,
+              className:
+                'entity-resizable-right-panel-container tag-resizable-panel-container',
+            }}
+          />
+        ),
+      },
+      {
+        label: (
+          <div data-testid="assets">
+            {t('label.asset-plural')}
+            <span className="p-l-xs ">
+              {getCountBadge(assetCount ?? 0, '', activeTab === 'assets')}
+            </span>
+          </div>
+        ),
+        key: 'assets',
+        children: (
+          <ResizablePanels
+            className="tag-height-with-resizable-panel"
+            firstPanel={{
+              className: 'tag-resizable-panel-container',
+              children: (
+                <AssetsTabs
+                  assetCount={assetCount}
+                  entityFqn={tagItem?.fullyQualifiedName ?? ''}
+                  isSummaryPanelOpen={Boolean(previewAsset)}
+                  permissions={MOCK_TAG_PERMISSIONS}
+                  ref={assetTabRef}
+                  type={AssetsOfEntity.TAG}
+                  onAddAsset={() => setAssetModalVisible(true)}
+                  onAssetClick={handleAssetClick}
+                  onRemoveAsset={handleAssetSave}
+                />
+              ),
+              ...COMMON_RESIZABLE_PANEL_CONFIG.LEFT_PANEL,
+            }}
+            hideSecondPanel={!previewAsset}
+            secondPanel={{
+              children: previewAsset && (
+                <EntitySummaryPanel
+                  entityDetails={previewAsset}
+                  handleClosePanel={() => setPreviewAsset(undefined)}
+                />
+              ),
+              ...COMMON_RESIZABLE_PANEL_CONFIG.RIGHT_PANEL,
+              className:
+                'entity-resizable-right-panel-container tag-resizable-panel-container',
+            }}
+          />
+        ),
+      },
+    ];
+
+    return items;
+  }, [
+    tagItem,
+    previewAsset,
+    activeTab,
+    assetCount,
+    assetTabRef,
+    handleAssetSave,
+    isDescriptionEditable,
+    editTagsPermission,
+    editDescriptionPermission,
+  ]);
+  const icon = useMemo(() => {
+    if (tagItem?.style?.iconURL) {
+      return (
+        <img
+          alt={tagItem.name ?? 'Tag Icon'}
+          className="align-middle object-contain"
+          data-testid="icon"
+          height={36}
+          src={tagItem.style?.iconURL}
+          width={32}
+        />
+      );
+    }
+
+    return <IconTag className="h-9" style={{ color: DE_ACTIVE_COLOR }} />;
+  }, [tagItem]);
+
+  useEffect(() => {
+    getTagData();
+    fetchClassificationTagAssets();
+  }, []);
+
+  useEffect(() => {
+    if (tagItem) {
+      fetchCurrentTagPermission();
+    }
+  }, [tagItem]);
+
+  if (isLoading) {
+    return <Loader />;
+  }
+
+  if (!tagItem) {
+    return null;
+  }
```

Comment:
i think if there is no data we should show no data placeholder?

Reply:
Updated the placeholder.